morseMessenger/flashHook.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { morseDictionary, DictOptions } from './morseDictionary';
+
+export default function useFlash() {
+  const TORCH_ON = 'torch';
+  const TORCH_OFF = 'off';
+  const PAUSE = 1000;
+  const FLASH_PAUSE = 400;
+  const FLASH_SHORT = 100;
+  const FLASH_LONG = 700;
+  const [flashMode, setFlashMode] = useState(TORCH_OFF);
+  const [buttonDisabled, setButtonDisabled] = useState(false);
+
+  function torchOffAfter(timeInMs: number) {
+    return new Promise((resolve, reject) => {
+      setTimeout(() => {
+        setFlashMode(TORCH_OFF);
+        resolve();
+      }, timeInMs);
+    });
+  }
+  function torchOnAfter(timeInMs: number) {
+    return new Promise((resolve, reject) => {
+      setTimeout(() => {
+        setFlashMode(TORCH_ON);
+        resolve();
+      }, timeInMs);
+    });
+  }
+  function short(): Promise<any> {
+    return new Promise((resolve, reject) => {
+      torchOnAfter(FLASH_PAUSE)
+        .then(() => torchOffAfter(FLASH_SHORT))
+        .then(resolve);
+    });
+  }
+
```

Comment:
this is nit and the reason why people use eslint 😄  -> you should try to keep your whitespace (empty lines) consistent as then your code looks clean and polished.

Right now it looks like this:
```ts
torchOffAfter
torchOnAfter
short

long

pause
```

I think it would be better to have it like this:
```
torchOffAfter
torchOnAfter

short
long
pause
```
or like this:
```
torchOffAfter
torchOnAfter
short
long
pause
```
or empty line between each function, it's all up to u, but it's nice to do it in a way that looks like you separated blocks of code on purpose, not added empty lines randomly 😛